client/src/java/com/zimbra/client/ZSearchParams.java

```diff
@@ -159,6 +162,15 @@ else if (s.startsWith("g") && TYPE_GAL.startsWith(s))
      */
     private boolean mInDumpster;
 
+    /**
+     * If false, items with the \Deleted tag set are not returned.
+     */
+    private boolean includeTagDeleted = false;
+
+    private ZimbraFetchMode resultMode = ZimbraFetchMode.NORMAL;
```

Comment:
There's a naming conflict here: there is already an "mFetch" field which refers to which messages in a conversation should be expanded. The fetch mode here corresponds to the "resultMode" attribute, so I called it that.